api/src/test/java/org/pmiops/workbench/interceptors/AuthInterceptorTest.java

```diff
@@ -150,6 +163,37 @@ public void preHandleGet_nonBearerAuthorization() throws Exception {
     verify(mockResponse).sendError(HttpServletResponse.SC_UNAUTHORIZED);
   }
 
+  @Test
+  public void preHandleGet_usingServiceAccountNotInDb() throws Exception {
```

Comment:
I'm not really understanding these test cases.  How does this demonstrate the SA is not in the DB?  Is "bob@fake" the same email as the SA?

Reply:
My mistake. I had not set up these tests correctly. They should be using the correct email address and be a little more readable now. Thanks for catching that.